File: blimp/chunks/lang/bench/btree.zig
const std = @import("std");
fn treeBuild(depth: i64) i64 {
    if (depth == 0) return 1;
    const left = treeBuild(depth - 1);
    const right = treeBuild(depth - 1);
    return left + right + 1;
}
pub fn main() void {
    std.debug.print("{d}\n", .{treeBuild(25)});
}

File: blimp/chunks/lang/bench/btree.rs
fn tree_build(depth: i64) -> i64 {
    if depth == 0 { return 1; }
    let left = tree_build(depth - 1);
    let right = tree_build(depth - 1);
    left + right + 1
}
fn main() {
    println!("{}", tree_build(25));
}
